TicTacToe.java

```diff
@@ -160,11 +170,11 @@ else if (playerStartChar == 'N') {
 			case 1: game = new MinMaxGame(height, width, playerStart);
 					break;
 			case 2: game = new MinMaxGameHashed(height, width, playerStart);
-					break;
+			 		break;
 			case 3: game = new AlphaBetaGame(height, width, playerStart);
-					break;
+			 		break;
 			case 4: game = new LearnedValueGame(height, width, playerStart);
-					break;
+			 		break;
```

Comment:
Do you need a default in your `switch` statement?

Reply:
It cannot possibly be anything else than the things listed.